cmake/modules/platform/os/win.cmake

```diff
@@ -22,29 +22,15 @@
 set(OMR_WINVER "0x501")
```

Comment:
Please delete this line, its not used anymore

Reply:
@youngar ok, I've deleted.